test/parallel/parallel.status

```diff
@@ -8,7 +8,11 @@ prefix parallel
 # https://github.com/nodejs/node/issues/23207
 test-net-connect-options-port: PASS,FLAKY
 # https://github.com/nodejs/node/issues/26401
-test-worker-prof: PASS,FLAKY
+# There was a problem with V8 GC which potentially could
```

Comment:
I think we should keep commentary on why tests are skipped in github, in this case, in the already linked issue https://github.com/nodejs/node/issues/26401 (doing so would be consistent with existing patterns). So, this information here would go in #26401, and also likely into the commit message (its already in the the PR). Specifically, having recently gone through all the statuses, I wouldn't find the V8 link useful. It wouldn't allow me to know what the current state of getting this patch backported into node.js is.  I'm not even sure if it would need backporting, maybe it will come in a new V8 update? This latter information is important, so it should all be either in #26401 or another issue that allows the state to be tracked. If the V8 fix is going to land in a V8 update, I don't think another nodejs/node issue is required. If it will need to be backported/cherry-picked/something after landing in V8, then I think there should be a nodejs/node issue to track this.

Reply:
I have made a comment in #26401. Backporting on V8 has to be done manually but I'm not sure when nodejs is going to pickup the latest from V8 (branch 7.5 or 7.6), when it does we can revert this change. I have removed the extra comments from the status file.